cypress/support/pages/home.pageObject.js

```diff
@@ -5,12 +5,12 @@ class HomePageObject extends PageObject {
 
   get usernameLink() {
     return cy.getByDataCy('username-link');
+    return this.getByDataCy('username-link');
   }
 
   assertHeaderContainUsername(username) {
     this.usernameLink
       .should('contain', username);
+        .should('contain', username);
```

Comment:
There is a duplicate line in the `assertHeaderContainUsername` method. Remove the second `.should('contain', username);` to avoid redundancy.